src/sections/SideBar/components/IconMaker/IconMaker.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import HomeOutlinedIcon from "@material-ui/icons/HomeOutlined";
+import PersonOutlineOutlinedIcon from "@material-ui/icons/PersonOutlineOutlined";
+import MailOutlineOutlinedIcon from "@material-ui/icons/MailOutlineOutlined";
+import NotesOutlinedIcon from "@material-ui/icons/NotesOutlined";
+import FacebookIcon from "@material-ui/icons/Facebook";
+import InstagramIcon from "@material-ui/icons/Instagram";
+import LinkedInIcon from "@material-ui/icons/LinkedIn";
+import GitHubIcon from "@material-ui/icons/GitHub";
+
+const IconMaker = (props) => {
+    const selectIcon = (index) => {
+      return index === 0 ? <HomeOutlinedIcon className={props.icon} />
```

Comment:
I prefer insted of use a numer use something else more descriptive as if `icons === "home" ` also define those icon names in a constant object where you can use it everywhere else. Please don't use ternary operators conncatenated like this as is a bad practice. Instead create an Iconst object that contains properties you can access with the icon, for example:

Reply:
```suggestion
constants file:

export const ICONS = {
home: "home",
person: "person",
....}

IconMaker:

const ICON_ELEMENTS = {
[ICONS.home] : <HomeOutlined.. >,
[ICONS.person]: <PersonOutlined...>
}

return ICON_ELEMENTS[receivedIcon]


```

Reply:
But I rather prefer conditional rendering so please do something like:
```suggestion
      return 
      <> 
      {receivedIcon === ICONS.home && <HomeIcon>}
      {receivedIcon === ICONS.person && <PersonIcon>}
      </>
```